ota-plus-web/app/reactapp/src/components/updates/createWizard/step2/UpdatesWizardDetailListItem.jsx

```diff
@@ -185,6 +186,11 @@ class UpdatesWizardDetailListItem extends Component {
             </Col>
           </Form>
         </Row>
+        <Row>
+          <Col span={12}>
+            <div className="information-text">{t('updates.creating.wizard.information_step2')}</div>
```

Comment:
We should check how many items we have in the list and show the message only for one item.

Reply:
done